controllers/pollController.js

```diff
@@ -75,4 +75,26 @@ const deleteSinglePoll = async (req, res) => {
     res.status(500).send(err)
   }
 }
-module.exports = { addNewPoll, getPolls, getSinglePoll, updateSinglePoll,deleteSinglePoll }
\ No newline at end of file
+const voteAPoll = async (req, res) => {
+  const pollId = req.params.id;
+  const markedOptionId = req.params.optionId;
+  try {
+    const pollAllOptions = await optionsModel.findAll({ where: { poll_id: pollId } })
+    for (let index = 0; index < pollAllOptions.length; index++) {
```

Comment:
This is wrong way to implement it, you are not checking the user , who vote it. Without checking the user, you can give this error